sonic_data_client/virtual_db.go

```diff
@@ -140,21 +204,36 @@ func v2rEthPortFieldStats(paths []string) ([]tablePath, error) {
 	var tblPaths []tablePath
 	if strings.HasSuffix(paths[KeyIdx], "*") {
 		for port, oid := range countersPortNameMap {
+			var oport string
+			if eport, ok := i2e_aliasMap[port]; ok {
+				oport = eport
+			} else {
+				log.V(2).Infof("%v dose not have an external alias", port)
+				oport = port
+			}
+
 			tblPath := tablePath{
-				dbName:       paths[DbIdx],
-				tableName:    paths[TblIdx],
-				tableKey:     oid,
-				field:        paths[FieldIdx],
-				delimitor:    separator,
-				jsonTableKey: port,
+				dbName:    paths[DbIdx],
+				tableName: paths[TblIdx],
+				tableKey:  oid,
+				field:     paths[FieldIdx],
+				delimitor: separator,
+				//jsonTableKey: port,
```

Comment:
remove comments.

Reply:
Again, unless there is there a good reason to keep the commented-out line, please remove.